### PR TITLE
Medic chems

### PR DESCRIPTION
**Pull request by jroinc**
## About The Pull Request

Allows medics to buy a selection of advanced chems using their points, and removed a few noobtrap items that they could get for free from the vendor right next to them.

Also the hyperzine changes that have already been testmerged are in here because I didn't know about branches, so I guess either close the old one or wait for it to get in/denied so I can fix this one ;_;

## Why It's Good For The Game

Currently, it is both possible, and easy to fully duplicate a medic's kit as a PFC. This is part of a few upcoming changes intended to buff medic roundstart gear, in preparation for a nerfing of the medevac bed.

## Changelog
:cl:
add: Added 4 syringe cases, each buyable for 15 points, containing a syringe, and respectively 2 60u bottles of meralyne, dermaline, meraderm mix, and ironsugar mix.
add: Added a bottle of "synthetic" neurotoxin for later addition to MD vendors to test their chem mixes ingame and up RSR gameplay.
del: Removed a few things from medic vendor that were available for free in prep. I'll get the rest later.
balance: swapped the burn medkit's bottle of dermaline for an oxy bottle. Derm should not be found on the map at roundstart.

also everything with the hyperzine rework I made earlier

/:cl:

### CODE REVIEW

File: code/game/objects/items/reagent_containers/glass/bottle.dm [hunk 1]
```diff
@@ -147,6 +147,12 @@
 	desc = "A small bottle. Contains hot sauce."
 	icon_state = "bottle3"
 	list_reagents = list(/datum/reagent/consumable/capsaicin = 60)
+	
+/obj/item/reagent_container/glass/bottle/dermaline
+	name = "\improper Dermaline bottle"
+	desc = "A small bottle. Contains dermaline - used as a potent treatment against burns."
+	icon_state = "bottle15"
+	list_reagents = list(/datum/reagent/medicine/dermaline = 60)	list_reagents = list(/datum/reagent/consumable/capsaicin = 60)
```
Comment (spookydonut) [inline code/game/objects/items/reagent_containers/glass/bottle.dm]:
```suggestion
	list_reagents = list(/datum/reagent/medicine/dermaline = 60, /datum/reagent/consumable/capsaicin = 60)
```

**Comment by spookydonut** [comment]
Conflicts with #3061, either remove the changes made in #3061 or wait for that to be merged first.

File: code/game/objects/items/reagent_containers/glass/bottle.dm (hunk 1 above)
Comment (factoryman942) [reply code/game/objects/items/reagent_containers/glass/bottle.dm]: spicy burn juice

**Comment by jroinc** [comment]
I'm waiting for 3061 to be merged first.

File: code/game/objects/items/reagent_containers/glass/bottle.dm (hunk 1 above)
Comment (jroinc) [reply code/game/objects/items/reagent_containers/glass/bottle.dm]: sonova. I knew I misclicked earlier, THAT's what it did.

**Comment by jroinc** [comment]
Tested locally, everything spawns as it's supposed to, full of the intended chems. Probably should change capitalization of some things, but it works.

**Comment by jroinc** [comment]
fixed, same as 3061, just waiting for that one.

**Comment by jroinc** [comment]
Hyperzine autoinjector lowered to 10 pts. down from 15, due to other things that cost points and have actual value actually being added

File: code/modules/reagents/chemistry_reagents/medical.dm [hunk 2]
```diff
@@ -618,6 +618,11 @@ datum/reagent/medicine/synaptizine/overdose_crit_process(mob/living/L, metabolis
 
 	return ..()
 
+//datum/reagent/medicine/hyperzine/on_mob_add(mob/living/L, metabolism)
```
Comment (spookydonut) [inline code/modules/reagents/chemistry_reagents/medical.dm]: delete this :P

**Pull request merged**